native/tools/build-config

```diff
@@ -57,10 +57,10 @@ program
   })
 
 program
-  .command('to-json <build_config_name>')
+  .command('to-json <build_config_name> <platform>')
   .description('outputs the specified build config as JSON')
-  .action(async buildConfigName => {
-    const buildConfig = loadBuildConfig(buildConfigName)
+  .action((buildConfigName, platform) => {
+    const buildConfig = loadBuildConfig(buildConfigName, platform)
```

Comment:
Does the programm exit here with exit code 1 if an exception is thrown?

Reply:
Yes, there is no async function here :)